src/providers/postgres/qgspostgresconn.h

```diff
@@ -183,6 +183,12 @@ class QgsPostgresConn : public QObject
     Q_OBJECT
 
   public:
+    /// @param shared allow using a shared connection. Should never be
```

Comment:
th eentry point to the db connection is connectDb (and in DB Manager the connect of the psycopg).
Would be more clear (at least to me) to have a single entry point (the QgsConnectionPool) that can also manage MT.